apiaudio/api_request.py

```diff
@@ -38,13 +36,15 @@ def _build_header(cls):
 
     @classmethod
     def _post_request(cls, json, url=None):
+        loop_status_code = cls.__dict__.get("loop_status_code")
```

Comment:
Do all cls. instances have "loop_status_code"?

Reply:
Not necessarily, that's why we're doing a .get() on its dictionary of attributes